packages/auth/lib/modular/index.d.ts

```diff
@@ -659,6 +689,273 @@ export function getAdditionalUserInfo(
  * Returns the custom auth domain for the auth instance.
  *
  * @param auth - The Auth instance.
- * @returns A promise that resolves with the custom auth domain.
+ * @returns {string} A promise that resolves with the custom auth domain.
```

Comment:
this returns `* @returns {Promise<string>}`, one day it should be just `string` but that is when RNFB adopts new architecture with sync API calls to native.